src/main/java/com/google/sps/servlets/PerspectiveRequest.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.servlets;
+ 
+import com.google.common.collect.ImmutableMap;
+ 
+/**
+ * This class represents the data needed to send a request
+ * to the Perspective API and will be used to turn the data
+ * into a JSON formatted string.
+ */
+public class PerspectiveRequest {
+    private ImmutableMap<String, String> comment;
+    private ImmutableMap<String, ImmutableMap<String, String>> requestedAttributes;
+ 
+    public PerspectiveRequest(String text) {
+        this.comment = ImmutableMap.of("text", text);
+        this.requestedAttributes = ImmutableMap.of("TOXICITY", ImmutableMap.of());
```

Comment:
What does the second parameter do?

Reply:
basically you choose an attribute that you want the comment to be scored on (I chose toxicity for simplicity but it can be changed or more added). then each attribute maps to a "configuration" object, but I didn't feel we needed to configure anything since all we wanted was the score, so you can leave the configurations blank

Reply:
I'm not sure if I understand your note on the configurations. If it's not in use, why have a map at all?

Reply:
it's the way the API wants the request formatted for some reason. the requestedAttributes field is a map from an attribute to a "configuration" object which they say is commonly just an empty map. https://github.com/conversationai/perspectiveapi/blob/master/2-api/methods.md#scoring-comments-analyzecomment - this is the format I'm following for the class